var/spack/repos/builtin/packages/py-melissa-core/package.py

```diff
@@ -37,11 +39,16 @@ class PyMelissaCore(PythonPackage):
     depends_on("py-python-rapidjson@1.8:", type="run")
```

Comment:
mpi4py should be exactly 3.1.3, not 3.1.3+

Reply:
Fixed it, thanks.